source/blog/2017-02-16-hyperconverged-ovirt-with-ovn.md

```diff
@@ -0,0 +1,355 @@
+---
+title: HyperConverged oVirt (GlusterFS) with OVN (OpenVSwitch + OpenFlow + Tunnels) provider
+author: stirabos
+tags: community, documentation, howto, glusterfs, gluster, hyperconverged, hyper-converged, hosted-engine, OVN, SDN, demo, cockpit
+date: 2017-02-16 21:00:00 CET
+comments: true
+published: false
+---
+
+How-to build a demo env for HyperConverged oVirt with OVN.
+
+In this tutorial I'm going to provide the instructions to deploy a test environment from scratch to try out the newest oVirt 4.1 features.
+The idea is to keep everything as simple as possible to let you simply deploy a demo env to get in touch with new features.
+A production deployment requires more cares: in particular for this demo environment we are going to:
+* use just /etc/hosts instead of relying a properly configured DHCP and DNS infrastructure which is instead strongly recommended for a production system
+* use a single network for management, storage and OVN transport: using separated network is strongly recommended for production systems; to achieve the best performance on the storage side we recommend a 10GbE
+The advantage of this simplistic approach is that in less than a couple of hours you are able to set up from scratch a  rich environment to get in touch with the new features of 4.1 without many hassles. 
+
+READMORE
+
+## Background
+
+* **oVirt** is free, open-source virtualization management platform. A typical deployment is composed by one or more virtualization hosts and a central engine that manages them. The engine can run on a VM hosted by the hosts that it's managing, this setup is called **hosted-engine**.
+* **GlusterFS** is a is free and open source scalable network file-system. Using common off-the-shelf hardware, you can create a large and distributed storage solution for your virtualization needs. If you run GlusterFS to save your VMs on the same hosts you are using to run them, you have an **HyperConverged setup**.
+* OVN, **Open Virtual Network**, is part of the Open vSwitch (OVS) project: it complements the existing capabilities of OVS to add native support for virtual network abstractions, such as virtual L2 and L3 overlays and security groups. Since release 4.1, oVirt ships an OVN provider to be used to consume and manage OVN networks from oVirt.
+* **Cockpit** is an **interactive web-based server admin interface**.
+
+In this tutorial we'll se how to deploy a test environment deploying oVirt hosted-engine in an hyper-converged way consuming OVN networks; the deployment will be mainly performed in a graphical way using Cockpit.
+
+## Prerequisites
+
+* Three hosts (or three VMs enabling the nested virtualization support on your physical host) called also nodes; each node should have a least 6 GB or RAM, two disks (20GB for local storage and 200GB for GlusterFS) and single nic (three nics with 10GbE for the storage network are strongly advised for production systems).
```

Comment:
"each node should have a least 6 GB or RAM" --> "each node should have at least 6 GB of RAM"